content/docs/command-reference/exp/run.md

```diff
@@ -69,27 +69,30 @@ checkpoints with DVC during your code or script runtime (similar to a logger).
 
 To do so, first mark stage `outs` with `checkpoint: true` in `dvc.yaml`. Having
 at least a checkpoint <abbr>output</abbr> is needed so that the experiment can
-later restart based on that output's last <abbr>cached</abbr> state.
+later continue from that output's last <abbr>cached</abbr> state.
 
-⚠️ Using the `checkpoint` field in `dvc.yaml` is only compatibly with
+⚠️ Using the `checkpoint` field in `dvc.yaml` is only compatible with
 `dvc exp run`, `dvc repro` will abort if any stage contains it.
 
-Then, in the corresponding code, either call the `dvc.api.make_checkpoint()`
-function (Python), or write a signal file (any programming language) following
-the same steps as `make_checkpoint()` — please refer to that reference for
-details.
+Then, in your code either call the `dvc.api.make_checkpoint()` function
+(Python), or write a signal file (any programming language) following the same
+steps as `make_checkpoint()` — please refer to its reference for details.
 
-Once this is setup, you can use `dvc exp run` to begin the experiment. When the
-process finishes or gets interrupted (e.g. with Ctrl + `C`), DVC will
+You can now use `dvc exp run` to begin the experiment. On this very first use,
+any `checkpoint` outputs are deleted before running the experiment (regardless
+of whether they have cached versions). When the process finishes or gets
+interrupted (e.g. with Ctrl + `C`), DVC will
 [apply](/doc/command-reference/exp/apply) the last checkpoint to the
 <abbr>workspace</abbr> (overwriting any further changes done by the stage).
 
-`dvc exp run` again will continue from this point (useful for interrupted runs).
-Use `--reset` to roll-back the workspace to `HEAD` and restart the whole
-experiment. Alternatively, you can use `--rev` to continue from a specific
-(previous) checkpoint.
+Following uses of `dvc exp run` will continue from this point (using the latest
+cached versions of all outputs). You can add a `--rev` to continue from a
+previous checkpoint instead (list them with `dvc exp show`). Or use `--reset` to
+start over (discards previous checkpoints and deletes `checkpoint` outputs, like
+the first `dvc exp run`) — useful for re-training ML models, for example.
 
-Note that `dvc exp show` displays checkpoints with a special branching format.
+> Note that after resetting checkpoint experiments, you may want to use `dvc gc`
+> to clean up unnecessary data in the cache.
```

Comment:
I don't think this note is really necessary, but I suppose it doesn't hurt anything either.

Reply:
Can't resetting checkpoints several times fill up the cache pretty quickly?

Reply:
Sure, but that just comes with running a lot of experiments in general, it's not really specific to checkpoints.

Reply:
🤺 you're right. Removed from here but made a slightly more prominent note about this in the general Description.